compiler-rt/lib/builtins/CMakeLists.txt

```diff
@@ -561,7 +561,7 @@ set(aarch64_SOURCES
 )
 
 if(COMPILER_RT_HAS_ASM_SME AND (COMPILER_RT_HAS_AUXV OR COMPILER_RT_BAREMETAL_BUILD))
```

Comment:
COMPILER_RT_HAS_ASM_SME tests if the compiler supports SME assembly, but not if the compiler supports the C/C++ level attributes. You'll need to add another check to the function which sets this variable and probably rename the variable to `COMPILER_RT_HAS_AARCH64_SME` to make it clear the compiler supports more than just the ASM. (and also to add "aarch64" to the name to avoid future name clashes with other meanings of 'sme')

Reply:
Done.